setup/web_server_configuration.rst

```diff
@@ -287,6 +287,13 @@ The **minimum configuration** to get your application running under Nginx is:
             # try to serve file directly, fallback to index.php
             try_files $uri /index.php$is_args$args;
         }
+        
+        # optionally disable falling back to php script for the asset directories
```

Comment:
```suggestion
        # optionally disable falling back to PHP script for the asset directories
```

Reply:
why does it say "optionally" if it's enabled by default? I also don't really see the value of adding it. It only prevents use-cases like having a route starting with /bundles.